src/components/results/candidate-results/index.js

```diff
@@ -1,16 +1,22 @@
+import PropTypes from "prop-types";
 import {Component} from "react";
+import Guide from "../../guide";
 import PersonalityArchetype from "./personality-archetype";
 import PersonalityDetails from "./personality-details";
 import PersonalityDimensions from "./personality-dimensions";
 import PersonalityTraits from "../dimension-based-results/personality-traits";
 
 export default class CandidateResults extends Component {
+  static propTypes = {
+    options: PropTypes.shape({perspective: PropTypes.string}).isRequired
+  }
   render() {
     return (
       <section>
         <PersonalityArchetype {...this.props} />
         <PersonalityDimensions {...this.props} />
         <PersonalityDetails {...this.props} />
+        {this.props.options.perspective === "thirdPerson" && <Guide {...this.props} />}
```

Comment:
I think we want to get perspective the same was as here, in case it's passed in differently

https://github.com/traitify/traitify-widgets/blob/485a8baff35917cff5ab5e7b5eea746c774ae16c/src/components/results/candidate-results/personality-dimension-details/index.js#L38